tests/integration/schema/reinitialize-associations-test.js

```diff
@@ -0,0 +1,38 @@
+import Mirage from 'ember-cli-mirage';
+import Model from 'ember-cli-mirage/orm/model';
+import Schema from 'ember-cli-mirage/orm/schema';
+import Db from 'ember-cli-mirage/db';
+import {module, test} from 'qunit';
+
+// Model classes are defined statically, just like in a typical app
+var User = Model.extend({
+  address: Mirage.hasMany()
+});
+var Address = Model.extend();
+
+module('Integration | Schema | reinitialize associations', {
+  beforeEach: function() {
+    this.db = new Db({
+      users: [],
+      addresses: []
+    });
+    this.schema = new Schema(this.db);
+
+    this.schema.registerModels({
+      address: Address,
+      user: User
+    });
+
+    this.schema.user.create({ id: 1, name: 'Link' });
+    this.schema.address.create({ id: 1, country: 'Hyrule', userId: 1 });
```

Comment:
`id`s are auto assigned by the db. Also you might have been testing something else here, but the "standard" way to create related models using schema will be either

```js
let user = this.schema.user.create({ name: 'Link' });
this.schema.address.create({ country: 'Hyrule', user: user });
```

```js
let user = this.schema.user.create({ name: 'Link' });
user.createAddress({ country: 'Hyrule' });
```